### DISCUSSION TITLE
`column_not_found` in Nushell

### DISCUSSION THREAD

**Discussion by azzamsa** [sxyazi/yazi#501] [1. Q&A] [answered]
Hi 👋.

Coming from https://yazi-rs.github.io/docs/quick-start/#changing-working-directory-when-exiting-yazi. I get the error below.

```bash
Error: nu::shell::column_not_found

  × Cannot find column
    ╭─[/home/azzamsa/.config/nushell/env.nu:46:1]
 46 │ def --env x [] {
 47 │     let tmp = $"($env.TEMP)(char path_sep)yazi-cwd." + (random chars -l 5)
    ·                  ──┬─ ──┬─
    ·                    │    ╰── cannot find column 'TEMP'
    ·                    ╰── value originates here
 48 │   # let tmp = (mktemp -t "yazi-cwd.XXXXX")
    ╰────
```

I don't think we need this complicated code, `mktemp` works fine for me.

```diff
- let tmp = $"($env.TEMP)(char path_sep)yazi-cwd." + (random chars -l 5)
+ let tmp = (mktemp -t "yazi-cwd.XXXXX")
```

Btw, where to make a PR for the documentation website?
I can't find the related code.

Thanks!

**Comment by sxyazi** [comment]
Well, it was `mktemp` indeed before, but there's no `mktemp` on Windows, so it was replaced in https://github.com/yazi-rs/yazi-rs.github.io/pull/18.

I want to know if Nushell has a simple way to cater to both Windows and Unix? I'm not familiar with it :)

As for the code, you can click "edit this page" link at the bottom of that document page.

**Reply by XYenon** [reply]
It looks like we can get system info by [`sys` command](https://www.nushell.sh/commands/docs/sys.html#frontmatter-title-for-system).

**Reply by lpnh** [reply]
I think we can stick with [mktemp](https://www.nushell.sh/commands/docs/mktemp.html), now that it's compatible with both Windows and Unix, after the Nushell [0.88.0 release](https://www.nushell.sh/blog/2023-12-12-nushell_0_88_0.html#new-commands-toc).

**Comment by lpnh** [comment, marked as answer]
I took the liberty to send a [PR](https://github.com/yazi-rs/yazi-rs.github.io/pull/27).

Congratz on the 0.2.0, by the way 😊

**Reply by sxyazi** [reply]
Merged, thank you! Let me close this discussion then.
